cs-amazon/src/test/java/io/cloudslang/content/amazon/execute/QueryApiExecutorTest.java

```diff
@@ -371,6 +388,84 @@ public void testRebootInstances() throws Exception {
     }
 
     @Test
+    public void testRunInstances() throws Exception {
+        toTest.execute(getCommonInputs("RunInstances", HEADERS, ""), getRunInstancesCustomInputs(), getRunInstancesEbsInputs(), getRunInstancesElasticIpInputs(), getRunInstancesIamInputs(), getRunInstancesInstanceInputs(), getRunInstancesNetworkInputs());
+
+        verify(amazonSignatureServiceMock, times(1)).signRequestHeaders(any(InputsWrapper.class), eq(getHeadersMap()),
```

Comment:
We don't pass the (InputsWrapper) as parameter to this function, and the values from it are tested that are passed correctly after this with eq(getQueryParamsMap("RunInstances")) .